src/Components/Notifications/NoticeBoard.tsx

```diff
@@ -55,13 +57,30 @@ export const NoticeBoard: any = () => {
     );
   } else {
     notices = (
-      <Card key="no-notice" className="my-4 rounded-lg">
-        <CardContent>
-          <div className="text-xl text-center semibold">
-            {t("no_notices_for_you")}
+      <div className=" m-auto max-w-sm rounded overflow-hidden shadow-lg ">
+        <img
+          className="w-full"
+          src={"/images/Notificationbell.png"}
+          alt="bell"
+        ></img>
+        <div className="px-6 py-4">
+          <div className="font-bold text-xl mb-2">
+            You Dont Have Any Notices
           </div>
-        </CardContent>
-      </Card>
+          <p className="text-gray-700 text-base">
+            Did you know: We could render a fun fact about health here
+          </p>
+        </div>
+
+        <div className="px-6 pt-4 pb-2">
+          <ButtonV2
+            className=" mx-28 bg-primary-500 text-white hover:bg-primary-700 text-white font-bold py-2 px-4 rounded-full"
```

Comment:
Avoid specifying custom designs when using `ButtonV2`. This is to keep designs consistent across and easy to maintain.

```suggestion
            className="mx-28"
```

Reply:
do u want me to send another PR for this @rithviknishad ?

Reply:
Nope, just letting you know for next time. I've already committed it. I've marked the PR as need for testing.